spec/src/lib.rs

```diff
@@ -99,7 +99,7 @@ pub struct Genesis {
     pub compact_target: u32,
     pub uncles_hash: H256,
     pub hash: Option<H256>,
-    pub nonce: u64,
+    pub nonce: u64, // u128 is not supported by https://github.com/alexcrichton/toml-rs
```

Comment:
We can use `U128` on `numext-fixed-uint` instead, it supports `serde` and accurately express u128